nextcord/client.py

```diff
@@ -248,6 +248,10 @@ class Client:
         Warning: While enabling this will prevent "ghost" commands on guilds with removed code references, rolling out
         to ALL guilds with anything other than a very small bot will likely cause it to get rate limited.
 
+    rollout_guild_ids: List[:class:`int`]
```

Comment:
I dislike the name here.

Reply:
as they work as defaults, `default_guild_ids` may work better

also dont forget the docstrings if this is changed

Reply:
`rollout_guild_ids` is consistent with the naming of the other slash kwargs, but I'm fine with `default_guild_ids` as well.